src/steamcompmgr.cpp

```diff
@@ -841,7 +841,7 @@ bool MouseCursor::getTexture()
 	// Assume the cursor is fully translucent unless proven otherwise.
 	bool bNoCursor = true;
 
-	uint32_t cursorDataBuffer[m_width * m_height] = {0};
+	uint32_t *cursorDataBuffer = (uint32_t *)calloc(m_width * m_height, sizeof(uint32_t));
```

Comment:
Given this is C++:

Why not just use a `std::vector` of size `m_width * m_height`?

`auto cursorData = std::vector<uint32_t>(m_width * m_height);`

This would match the rest of this file.

Reply:
Fair enough, done